rackcity/views/pdu_views.py

```diff
@@ -0,0 +1,74 @@
+# from django.http import HttpResponse, JsonResponse
+from django.http import JsonResponse
+from rackcity.models import (
+    Rack,
+    Asset
+)
+from rackcity.api.serializers import (
+    serialize_power_connections,
+)
+# from rackcity.utils.log_utils import (
+#     log_action,
+#     log_bulk_import,
+#     log_delete,
+#     Action,
+#     ElementType,
+# )
+from rest_framework.decorators import permission_classes, api_view
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+# from rest_framework.parsers import JSONParser
+from http import HTTPStatus
+from rackcity.views.rackcity_utils import (
+    validate_asset_location,
+    validate_location_modification,
+    no_infile_location_conflicts,
+    records_are_identical,
+    get_sort_arguments,
+    get_filter_arguments,
+)
+import re
+import requests
+
+pdu_url = 'http://hyposoft-mgt.colab.duke.edu:8000/pdu.php?pdu=hpdu-rtp1-' # Need to specify rack + side in request, e.g. for A1 left, use A01L
```

Comment:
Ours is 8005 - do we need to use 8000 to test?

Reply:
Using 8000 for testing purposes bc 8005 doesn't work when I run locally. I suppose we should change it now and see what happens on dev